dev-client/src/translations/en.json

```diff
@@ -683,5 +650,9 @@
         "LINEAR": "Linear"
       }
     }
+  },
+  "image": {
+    "use_camera": "Use Camera",
+    "choose_from_gallery": "Choose From Gallery"
```

Comment:
```suggestion
    "use_camera": "Take Photo",
    "choose_from_gallery": "Photo Library"
```

Reply:
feedback for copy best lives in the figma file